topostats/processing.py

```diff
@@ -414,11 +416,19 @@ def run_dnatracing(  # noqa: C901
                     **dnatracing_config,
                 )
                 tracing_stats[direction] = tracing_results["statistics"]
-                ordered_traces = tracing_results["ordered_traces"]
-                cropped_images = tracing_results["cropped_images"]
+                ordered_traces = tracing_results["all_ordered_traces"]
+                cropped_images: dict[int, np.ndarray] = tracing_results["all_cropped_images"]
                 image_spline_trace = tracing_results["image_spline_trace"]
                 tracing_stats[direction]["threshold"] = direction
 
+                grain_trace_data[direction] = {
```

Comment:
Not for this PR but I wonder if we could simplify this in the future and have `dnatracing.trace_image()` return the dictionary that we want/need rather than cobbling it together here. :thinking:

Reply:
Agree!